### PR TITLE
Update getting started resources for enterprise catalog

### LINKED ISSUE #2282: Getting started links need to be updated for OCP
The documentation link may be pointing to the wrong doc. We should decide which of the other links we want to be there.

The documentation link needs to be pulled from the openshift constants if its not already so that it correctly points to customized doc links, such as for Online.

### PR DESCRIPTION
**Pull request by spadgett**
Closes #2282

screen shot 2017-10-25 at 1 38 27 pm

@sspeiche 
/assign @jwforres

### CODE REVIEW

File: app/scripts/constants.js [hunk 1]
```diff
@@ -64,6 +64,33 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
...
+      {
+        title: 'Documentation',
+        help: ''
+      },
+      {
+        title: 'Interactive Learning Portal',
+        href: 'https://learn.openshift.com'
+      },
+      {
+        title: 'Red Hat Container Development Kit',
```
Comment (jwforres) [inline app/scripts/constants.js]: feels a bit wordy compared to the other links, can we drop the "Red Hat" from it?  @sspeiche ?

Comment (spadgett) [reply app/scripts/constants.js]: > feels a bit wordy compared to the other links, can we drop the "Red Hat" from it?

nudge @sspeiche :)

**Comment by spadgett** [comment]
@jwforres updated

**Comment by spadgett** [comment]
openshift web console 2017-10-30 14-09-07
